packages/staking/src/features/BrowsePools/StakePoolCard/StakePoolCard.tsx

```diff
@@ -0,0 +1,36 @@
+import { Card, Flex, Text } from '@lace/ui';
+import cn from 'classnames';
+import { PoolMetric } from './PoolMetric';
+import { StakePoolCardProgressBar } from './SkatePoolCardProgressBar';
+import * as styles from './StakePoolCard.css';
+import { MetricType } from './types';
+
+export interface StakePoolCardProps {
+  title?: string;
+  metricType: MetricType;
+  metricValue?: number;
+  saturation: number;
+  selected?: boolean;
+  onClick?: () => void;
+}
+
+export const StakePoolCard = ({
+  title = '-',
+  metricType,
+  metricValue,
+  saturation,
+  selected,
+  onClick,
+}: StakePoolCardProps) => (
+  <Card.Outlined className={cn(styles.card, selected && styles.cardSelected)} {...(onClick ? { onClick } : {})}>
```

Comment:
Spread is redundant, you can assign it with `undefined` and it will have the same effect.

```suggestion
  <Card.Outlined className={cn(styles.card, selected && styles.cardSelected)} onClick={onClick}>
```